wildfly-getting-started-archetype/src/main/resources/archetype-resources/src/test/java/org/wildfly/security/examples/test/SampleIT.java

```diff
@@ -0,0 +1,44 @@
+#set( $symbol_pound = '#' )
+#set( $symbol_dollar = '$' )
+#set( $symbol_escape = '\' )
+package org.wildfly.security.examples.test;
```

Comment:
@darranl  package is not correct.I fixed it in https://github.com/darranl/wildfly-archetypes/pull/2